src/layer/DivOverlay.js

```diff
@@ -19,16 +19,16 @@ export var DivOverlay = Layer.extend({
 	// @aka DivOverlay options
 	options: {
 		// @option offset: Point = Point(0, 7)
-		// The offset of the popup position. Useful to control the anchor
+		// The offset of the overlay position. Useful to control the anchor
 		// of the popup when opening it on some overlays.
 		offset: [0, 7],
 
 		// @option className: String = ''
-		// A custom CSS class name to assign to the popup.
+		// A custom CSS class name to assign to the overlay.
 		className: '',
 
 		// @option pane: String = 'popupPane'
-		// `Map pane` where the popup will be added.
+		// `Map pane` where the overlay will be added.
 		pane: 'popupPane'

```

Comment:
Shouldn't we change option default value? May be unset?

Reply:
if it is a abstract class (like you said), then I would say yes